content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -93,6 +93,371 @@ service:
       exporters: [datadog/api]
 ```
 
+### Environment Specific Setup
+
+#### Host:
+
+- Download the appropriate binary from [the project repository latest release](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/latest).
+
+- Create a `otel_collector_config.yaml` file. Here is an example template to get started. It enables the collector's otlp receiver and datadog exporter.
+
+- Run on the host with the configration yaml file set via the `--config` parameter. For example,
+
+  ```
+    curl -L https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/latest/download/otelcontribcol_linux_amd64 | otelcontribcol_linux_amd64 --config otel_collector_config.yaml
+  ```
+
+#### Docker
+
+Run an Opentelemetry Collector container to receive traces either from the [installed host](#receive-traces-from-host), or from [other containers](#receive-traces-from-other-containers).
+
+##### Receive Traces From Host
+
+- Create a `otel_collector_config.yaml` file. [Here is an example template](https://docs.datadoghq.com/tracing/setup_overview/open_standards/#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's otlp receiver and datadog exporter.
+
+- Use a published docker image such as [`otel/opentelemetry-collector-contrib:latest`](https://hub.docker.com/r/otel/opentelemetry-collector-contrib/tags)
+
+- OpenTelemetry Traces are sent to the OpenTelemetry Collector over TCP or UDP on a number of ports, which must be exposed on the container.  By default traces are sent over `OTLP/gRPC on port 55680`, but common protocols and their ports include: 
+
+  ```
+  Zipkin/HTTP on port 9411
+  Jaeger/gRPC on port 14250
+  Jaeger/HTTP on port 14268
+  Jaeger/Compact on port 6831 (UDP)
+  OTLP/gRPC on port 55680
+  OTLP/HTTP on port 55681
+  ```
```

Comment:
I think we should rely on official OpenTelemetry docs here. This is not Datadog-specific and it is more likely to be up to date on the official docs.

Reply:
I couldn't find an organized list within their docs. This would also fall into the bucket of, happy to try to fix upstream but would prefer to leave this in our docs for now until upstream is in a better place